StortSpelProjekt/Engine/src/Physics/Physics.cpp

```diff
@@ -0,0 +1,16 @@
+#include "stdafx.h"
+#include "Physics.h"
+
+Physics::Physics()
+{
+	
+}
+
+Physics::~Physics()
+{
+}
+
+bool Physics::CheckOBBCollision(DirectX::BoundingOrientedBox obb1, DirectX::BoundingOrientedBox obb2)
+{
+	return obb1.Intersects(obb2);
+}
```

Comment:
Send address instead of object. (but first check if the size is less then 8 bytes)

Reply:
fixed, does not appear to be used yet.